install_poetry/install_poetry.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+#  Copyright (c) 2022 Ecco Sneaks & Data
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -Eeuo pipefail
+
+curl -sSL https://install.python-poetry.org | python3 - --preview
+export PATH=/github/home/.local/bin:$PATH
```

Comment:
Has this been tested in containers? Because I remember the path is different if a build is run inside a container.

Reply:
Could you please provide an example? I did not found any build where PATH would be different. I also tested it in the build of metadata-manager, and it seems to work correctly: https://github.com/SneaksAndData/metadata-manager/runs/6965162887?check_suite_focus=true

Reply:
looking at your example i think all is good, consider resolved.